src/sentry/static/sentry/app/views/settings/project/projectServiceHookDetails.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createReactClass from 'create-react-class';
+import {Box} from 'grid-emotion';
+
+import {t} from '../../../locale';
+import {
+  addErrorMessage,
+  addSuccessMessage,
+} from '../../../actionCreators/settingsIndicator';
+import AsyncView from '../../asyncView';
+import ApiMixin from '../../../mixins/apiMixin';
+import Form from '../components/forms/form';
+import JsonForm from '../components/forms/jsonForm';
+import serviceHookSettingsFields from '../../../data/forms/serviceHookSettings';
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const TOAST_DURATION = 10000;
```

Comment:
this was copy pasted from org settings

Reply:
Toasts now have a default duration configured in a constants file